src/main/java/Sinep.java

```diff
@@ -0,0 +1,47 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class Sinep {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String greeting = "Hello! I'm Sinep, your personal chatbot!\n"
```

Comment:
Indentation for wrapped lines is correct (8 spaces) 👍 other indentations are still 4 spaces, was tab possibly used?